packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts

```diff
@@ -8,29 +8,51 @@
 // tslint:disable:no-unused-expression
 
 import { expect } from 'chai';
-import * as path from 'path';
-import * as shell from 'shelljs';
-import { workspace } from 'vscode';
-import { JAVA_HOME_KEY, JAVA_MEMORY_KEY } from '../../src/requirements';
+import { createSandbox, SinonSandbox, SinonStub } from 'sinon';
+import * as vscode from 'vscode';
+import { JAVA_HOME_KEY, resolveRequirements } from '../../src/requirements';
+import pathExists = require('path-exists');
+import * as cp from 'child_process';
+
+const runtimePath = '~/java_home/real/jdk/openjdk1.8.0.302_8.56.0.22_x64';
```

Comment:
Yes, that would be the hope! I don't have a windows machine, so wasn't able to test that path. If the stubs weren't present, it failed either way since we attempt to validate that that path exists, and it doesn't with vs code.